aws/aws-how-to/instances/automatically-setup-multiple-nics.rst

```diff
@@ -0,0 +1,55 @@
+Set up automated network configuration for instances with multiple NICs
+=======================================================================
+
+Ubuntu instances running on AWS can be configured to automatically configure source routing on multiple NICs via cloud-init, starting with `v24.1`_, on different `events`_, as:
+
+* First boot (enabled by default)
+* Subsequent boots
+* Hotplug NIC additions and removals
+
+.. note::
+
+   On Ubuntu Noble and newer, hot-plugged NICs are configured by default.
+
+Enable additional network configuration events
+----------------------------------------------
+
+To enable network configuration on additional `events`_, one can add supplemental user-data at instance creation time:
+
+..  code-block:: yaml
+
+    #cloud-config
+    # apply network config on every hotplug event
+    updates:
+      network:
+        when: ['hotplug']
+
+..  code-block:: yaml
+
+    #cloud-config
+    # apply network config on every boot and hotplug event
+    updates:
+      network:
+        when: ['boot', 'hotplug']
+
+Enable network configuration on hotplug events on running instances
+-------------------------------------------------------------------
+
+To enable network configuration on hotplug events on already running instances, execute the following command in the instance:
+
+.. code-block::
+
+    $ cloud-init devel hotplug-hook -s net enable
+
+This is a last resort command for administrators to enable hotplug in running instances.
+The recommended method to enable this is using user-data while creating the instance as described previously.
+For more information about this command, see `cloud-init's CLI documentation`_.
+
+Further references
```

Comment:
If we add the reference in the previous section, this section can be deleted.

Reply:
Done.